src/licensedcode/data/licenses/dbad.yml

```diff
@@ -0,0 +1,7 @@
+key: dbad
+name: Dont Be A Dick Public License
+short_name: dbad
+category: Public

```

Comment:
The category should be ` Proprietary Free`

Reply:
But its a public license still should I write there Proprietary Free?????